arangosh/Benchmark/BenchFeature.cpp

```diff
@@ -451,31 +554,40 @@ bool BenchFeature::writeJunitReport(BenchRunResult const& result) {
   return ok;
 }
 
-void BenchFeature::printResult(BenchRunResult const& result) {
+void BenchFeature::printResult(BenchRunResult const& result, VPackBuilder& builder) {
   std::cout << "Total request/response duration (sum of all threads): " << std::fixed
             << result.requestTime << " s" << std::endl;
-
+  builder.add("requestTime", VPackValue(result.requestTime));
   std::cout << "Request/response duration (per thread): " << std::fixed
             << (result.requestTime / (double)_concurrency) << " s" << std::endl;
+  builder.add("requestResponseDurationPerThread", VPackValue(result.requestTime / (double)_concurrency));
 
   std::cout << "Time needed per operation: " << std::fixed
-            << (result.time / _realOperations) << " s" << std::endl;
-
+            << (result.time / _operations) << " s" << std::endl;
+  builder.add("timeNeededPerOperation", VPackValue(result.time / _realOperations));
```

Comment:
The line above uses `result.time / _operations`, but here we add `result.time / _realOperations`.

Reply:
good spot!